src/lib/comms/sol-socket-dtls-impl-tinydtls.c

```diff
@@ -655,6 +656,98 @@ get_psk_info(struct dtls_context_t *ctx, const session_t *session,
     return r;
 }
 
+static int
+get_ecdsa_key(struct dtls_context_t *ctx, const session_t *session,
+    const dtls_ecdsa_key_t **result)
+{
+    struct sol_socket_dtls *socket = dtls_get_app_data(ctx);
+    struct sol_network_link_addr addr;
+    static unsigned char ecdsa_priv_key[SOL_DTLS_ECDSA_PRIV_KEY_LEN],
```

Comment:
What out for this static variable. Image what happens if you have two sockets on the same process using RPK....